src/packagedcode/models.py

```diff
@@ -117,14 +117,6 @@ class AssertedLicense(Model):
     notice = StringType(default=None)
     url = StringType(default=None)
 
-    def as_dict(self):
```

Comment:
@pombredanne  in order to override the default constructor in `schematics` we need to edit the `Model` class in `schematics.models.py`. But everytime we configure `scancode` we get a fresh install of `schematics